packages/core/src/actions/invoke.ts

```diff
@@ -53,6 +55,23 @@ function resolve(
             })
           : configuredInput
     });
+
+    if (subscribe) {
+      actorRef.subscribe({
+        next: (snapshot) => {
+          if (actorContext.self.status === ActorStatus.Running) {
+            actorContext.self.send({
+              type: `xstate.snapshot.${id}`,
+              id,
+              snapshot
+            });
+          }
+        },
+        error: () => {
+          /* TODO */
+        }
```

Comment:
that shouldn't be here as if we are going to add an error listener to the `actorRef` then it will be done outside of this if block

Reply:
Not sure how to resolve this; removing this causes a test failure

Reply:
I'm not 100% sure what I was thinking about here 😅 I had some concerns related to this right now but I think they are addressed, see this test case:
https://github.com/statelyai/xstate/blob/23402e34c178ba9a000e22c54765090b8fcfc52f/packages/core/test/errors.test.ts#L213